backend/src/services/calendar/core/serializers.py

```
@@ -0,0 +1,918 @@
+
+
+# ============================================================================
+# EVENT SERIALIZER
+# ============================================================================
+
+
+def serialize_event(
+    event: Event,
+    user_email: Optional[str] = None,
+    max_attendees: Optional[int] = None,
+    time_zone: Optional[str] = None,
```

Ruff (0.14.13)

Unused function argument: `time_zone`

(ARG001)